tests/contrib/test_kubernetes/conftest.py

```diff
@@ -0,0 +1,152 @@
+import os
+
+import pytest
+from kubernetes import client, config
+
+from tests.conftest import long
+
+
+def _cut_empty_lines(string):
```

Comment:
use from docker tests? maybe move to conftest

Reply:
moved to `conftest` for the whole test suite